installer/nw-chromedriver-version.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+
+/**
+ * Finds the closest nw-chromedriver module and returns the version
+ */
+module.exports = function() {
+  return findNwChromeDriver()
+    .then(function(modulePath) {
+      return readJSON(path.join(modulePath, "package.json"));
+    })
+    .then(function(json) {
+      return json.version;
+    });
+};
+
+function findNwChromeDriver() {
+  const searchPath = path.resolve(path.join(__dirname, ".."));
+
+  return new Promise(function(resolve, reject) {
+    const readdir = function(searchPath) {
```

Comment:
I don't mind all that much, as it's fairly localised, but perhaps consider whether there is too much shadowing going on here, with `fs.readdir` and `readdir`, and `searchPath` in two different scopes. Potential confusion for future self?